### PR TITLE
basic logging config

### PR DESCRIPTION
**Pull request by rekibnikufesin**
Resolves CAPI #18

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -15,6 +16,10 @@
 # whatever the makefile says...
 app.config.from_envvar('ENV_CONFIG_FILE')
 
+# setup logging
+fileConfig('logging.config')
```
Comment (robrobbins) [inline app.py]: let's put logging into the `config` folder alongside the ENV var ones. there's gunna be other .config files and i don't want them littering up the root dir

File: core/aws.py [hunk 2]
```diff
@@ -1,29 +1,23 @@
 import json
 import boto3
 from botocore.exceptions import ClientError
+from app import app
+import logging
+
+logging.config.fileConfig('logging.config')
+log = logging.getLogger()
+log.setLevel(app.config['LOG_LEVEL'])
```
Comment (robrobbins) [inline core/aws.py]: i dislike this whole block in this file. kind of a chicken vs egg smell here. it's cuz the way we use secrets to set ENV vars. this one i'd just leave the way it was and let it raise that clientError.

i dont like this whole circle dance in this file

File: core/dynamo.py [hunk 3]
```diff
@@ -7,6 +7,10 @@
 from flask import current_app, g
 import boto3
 # from botocore.exceptions import ClientError
+import logging.config
+
+logging.config.fileConfig('logging.config')
+log = logging.getLogger()
```
Comment (robrobbins) [inline core/dynamo.py]: why? `current_app` is available here. `set_dynamo_table` happens in the `request` lifecylce. shouldn't all this be done already?

File: core/protocol.py [hunk 4]
```diff
@@ -9,6 +9,10 @@
 from web3.middleware import geth_poa_middleware
 from computable.contracts import Voting, Datatrust, Listing
 from computable.helpers.transaction import call, transact, send
+import logging.config
+
+logging.config.fileConfig('logging.config')
+log = logging.getLogger()
```
Comment (robrobbins) [inline core/protocol.py]: again. why not `current_app.logger` ?

File: core/s3.py [hunk 5]
```diff
@@ -5,6 +5,10 @@
 """
 from flask import current_app, g
 import boto3
+import logging.config
+
+logging.config.fileConfig('logging.config')
+log = logging.getLogger()
```
Comment (robrobbins) [inline core/s3.py]: i herd there's this thing `current_app.logger`...

am i wrong on these?

**Review by robrobbins — CHANGES_REQUESTED** [review]

File: core/s3.py (hunk 5 above)
Comment (rekibnikufesin) [reply core/s3.py]: Nope. You are correct. Completely overlooked that on my part.

File: core/aws.py (hunk 2 above)
Comment (rekibnikufesin) [reply core/aws.py]: Reverted

**Review by robrobbins — APPROVED** [review]

**Pull request merged**